assertj-core/src/test/java/org/assertj/core/internal/strings/Strings_assertEqualsNormalizingWhitespace_Test.java

```diff
@@ -80,6 +80,10 @@ public static Stream<Arguments> equalNormalizingWhitespaceGenerator() {
                      Arguments.of(" my\tfoo bar ", " my\tfoo bar "),   // same
                      Arguments.of(null, null),   // null
                      Arguments.of(" \t \t", " "),
-                     Arguments.of(" abc", "abc "));
+                     Arguments.of(" abc", "abc "),
+                     Arguments.of("my\u00A0foo bar", "my foo bar"), // non-breaking space
+                     Arguments.of("my\u2007foo bar", "my foo bar"), // non-breaking space
+                     Arguments.of("my\u202Ffoo bar", "my foo bar")); // non-breaking space
+
```

Comment:
> I think that the breaking-space management could be improved by using a public constant so that we don't have to update all the tests if some of the breaking space codes change over time.

Feel free to pull constants into `StringsBaseTest`, similar to what `MapsBaseTest` does with map instances.

Reply:
I've done that in an other commit :+1: